--- a/node_modules/bin-build/index.js
+++ b/node_modules/bin-build/index.js
@@ -5,7 +5,7 @@ const execa = require('execa');
 const pMapSeries = require('p-map-series');
 const tempfile = require('tempfile');
 
-const exec = (cmd, cwd) => pMapSeries(cmd, x => execa.shell(x, {cwd}));
+const exec = (cmd, cwd) => pMapSeries(cmd, x => execa.shell(x, {cwd, shell: process.env.SHELL}));
 
 exports.directory = (dir, cmd) => {
 	if (typeof dir !== 'string') {
